commands/host/rabbitmq

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#ddev-generated
+
+## Description: A host wrapper command for rabbitmq/rabbitmq to enhance the command with "launch" capabilities
+## Usage: rabbitmq
+## Example: ddev rabbitmq
+
+CMD=$1
+
+launch_rabbitmq_gui() {
+    echo "Launching RabbitMQ Management UI..."
+    echo "Login using 'rabbitmq' user and 'rabbitmq' password."
+
+    rabbitmq_port=$(ddev exec "yq '.services.rabbitmq.expose[0]' .ddev/docker-compose.rabbitmq.yaml")
```

Comment:
HTTP_EXPOSE and HTTPS_EXPOSE should be used here (because `expose` returns the port for HTTP only):

```suggestion
    PROTOCOL="HTTP"
    if [ "${DDEV_PRIMARY_URL%://*}" != "http" ] && [ -z "${GITPOD_WORKSPACE_ID:-}" ] && [ "${CODESPACES:-}" != "true" ]; then
      PROTOCOL="HTTPS"
    fi
    rabbitmq_port=$(ddev exec "yq '.services.rabbitmq.environment[] | select(contains(\"${PROTOCOL}_EXPOSE\")) | split(\":\")[0] | split(\"=\")[1]' .ddev/docker-compose.rabbitmq.yaml")
```